scripts/tick-cluster.js

```diff
@@ -527,10 +547,19 @@ function send(host, arg1, arg2, arg3, callback) {
 
 program
     .version(require('../package.json').version)
-    .arguments('<size>')
+    .option('-i, --interpreter [interpreter]')
+    .option('-n [number of ringpop nodes]')
+    .arguments('<program> [args...]')
     .description('tick-cluster is a tool that launches a ringpop cluster of arbitrary size')
-    .action(function onAction(size) {
-        procsToStart = +size;
+    .action(function onAction(path, args, options) {
+        programInterpreter = options.interpreter;
+        if (options.N)
+            procsToStart = parseInt(options.N);
+        programPath = path;
+        if (programPath[0] !== '/') {
+            programPath = './' + programPath;
+        }
+        programArgs = args;
     });
```

Comment:
size is an option now (default if 5)

program path is prepended with ./ in case it is a relative path. We do this because to launch a binary in the current directory we need to type "./program". Just typing "program" would not start a process. This has the added bonus that we don't look in the path for binaries which could lead to some unexpected errors.